src/ucp/tag/tag_match.h

```diff
@@ -10,12 +10,16 @@
 #include <ucp/api/ucp_def.h>
 #include <ucp/core/ucp_types.h>
 #include <ucs/datastruct/queue_types.h>
+#include <ucs/datastruct/khash.h>
 #include <ucs/sys/compiler_def.h>
 
 
 #define UCP_TAG_MASK_FULL     0xffffffffffffffffUL  /* All 1-s */
 
 
+KHASH_MAP_INIT_INT64(ucp_tag_offload_hash, ucp_worker_iface_t *);
```

Comment:
need to declare a map using ucp_tag_t rather than int64_t, or assert that the types are the same (which is not 100% true..)